src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReleaseApprovalServiceTests.cs

```diff
@@ -1785,7 +1785,6 @@ await using (var context = InMemoryApplicationDbContext(contextId))
                 var saved = await context.Releases.Include(r => r.ReleaseStatuses)
```

Comment:
I think you could remove this whole block now as it's not really doing anything. There's other tests checking the release status has been saved correctly, so I think this can go.

```
await using (var context = InMemoryApplicationDbContext(contextId))
{
    var saved = await context.Releases.Include(r => r.ReleaseStatuses)
        .FirstAsync(r => r.Id == release.Id);
}
```